src/utils.js

```diff
@@ -173,3 +173,47 @@ export function promptForName() {
   } while (!(username && username.length));
   return username;
 }
+
+export function angleToDirection(angle) {
```

Comment:
We should probably just ditch this in favor of the `angleTo4Direction`, it makes for cleaner event names anyway (avoids the "downdown").

Also I know I wrote the original function but I can't help but feel there is a much cleaner way to do this hehe.